src/core/ext/transport/chttp2/transport/flow_control.cc

```diff
@@ -36,11 +36,16 @@
 #include <grpc/support/log.h>
 
 #include "src/core/lib/gpr/useful.h"
+#include "src/core/lib/gprpp/global_config_env.h"
 #include "src/core/lib/iomgr/exec_ctx.h"
 #include "src/core/lib/resource_quota/memory_quota.h"
 
 grpc_core::TraceFlag grpc_flowctl_trace(false, "flowctl");
 
+GPR_GLOBAL_CONFIG_DEFINE_BOOL(
```

Comment:
Would it be worth noting that this only works if a resource quota is set?

Reply:
done